docs/src/modules/components/ApiPage/sections/ToggleDisplayOption.tsx

```diff
@@ -20,6 +22,10 @@ export const API_LAYOUT_STORAGE_KEYS = {
 } as const;
 
 const getRandomOption = () => {
+  if (/Algolia Crawler/.test(navigator.userAgent) || /Googlebot/.test(navigator.userAgent)) {
```

Comment:
We should probably be less specific in bot detection, just to not lose SEO points on bing/yandex/baidu/...

e.g. use a regex like: https://stackoverflow.com/a/20084661

Reply:
```suggestion
  if (/bot|googlebot|crawler|spider|robot|crawling/i.test(navigator.userAgent)) {
```